### PR TITLE
Fixing formatting throw in the project

### LINKED ISSUE #201: Fix formatting issue and correct names of files throughout the project.
## Description

- Run Swiftformat and fix the formatting in the `ObfuscatorError` file.
- Run Swiftformat and fix formatting in all files in project, if it's needed.

### PR DESCRIPTION
**Pull request by iBYarchitect**
## Summary
This PR is a part of the work to resolve #201 

## Changes
- Fixing formatting in ObfuscatorError.swift

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] I've performed a self-review of my code.
- [x] I've added tests for my code (if it makes sense).
- [x] I've updated `CHANGELOG.md` file.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
  - Update PR template [#194](https://github.com/ios-course/ironfoudation-team-project/pull/194)
  - Update revision of the `SimpleNetworkService` package in the project - [#204](https://github.com/ios-course/ironfoudation-team-project/pull/204)
+ - Fixing issues after running SwiftFormat and SwiftLint through the project [#206](https://github.com/ios-course/ironfoudation-team-project/pull/206)
```
Comment (kirpichonok) [inline CHANGELOG.md:24]: nit: 
I'm not sure that we need to add this info into CHANGELOG.md.

**Review by kirpichonok — COMMENTED** [review]
WDYT?

**Review by kirpichonok — APPROVED** [review]
LGTM

**Pull request merged**